addons/portal/wizard/portal_wizard.py

```diff
@@ -66,131 +47,8 @@ def onchange_portal_id(self, cr, uid, ids, portal_id, context=None):
                     }))
         return {'value': {'user_ids': user_changes}}
```

Comment:
line no: 42 to 48 should be improved. like this : 
 in_portal = portal_id in contact.user_ids[0].groups_id.ids
return self.user_ids = user_changes